index.html

```diff
@@ -29,7 +29,14 @@
     <div id="showMoreLessLanguages"></div>
     <div id="username_selector"></div>
```

Comment:
I think we shouldn't show the usernames list unless the user wants to see it. You should remove it and another button in `Settings` to show / hide username list in the main page.